### DISCUSSION TITLE
Failed to Download Starter Kit Error

### DISCUSSION THREAD

**Discussion by CynthiaBlue** [quasarframework/quasar#11193] [CLI - PWA mode] [answered]
I have a new MacBook and am installing Quasar.  I used Homebrew to install Node and have Node Version 16.13.0 and NPM version 8.1.0 installed.  I used Homebrew because when I tried just installing Node via the download package I was getting permission errors when installing Quasar.

However, now when I try to create a new Quasar Project using 

```quasar create new_dir```

I get the following error message:
```Quasar CLI · Failed to download repo quasarframework/quasar-starter-kit#master: EPERM: operation not permitted, utime '/Users/mymac/.quasar-starter-kits/quasarframework-quasar-starter-kit#master/.github'```

From what I'm seeing online it's possible that the github download site to install the starter kit is temporarily down? Could that be the case? Or is there something else I'm hitting? 
Thanks.

This is my quasar info output
<img width="420" alt="Screen Shot 2021-11-02 at 6 39 45 PM" src="https://user-images.githubusercontent.com/34698014/139970376-2fa0246a-6985-4c52-b603-3751a190d3ef.png">
:

**Comment by MilosPaunovic** [comment]
This seems to be an error with your local environment and/or VSCode. Check this thread out:
[https://github.com/Microsoft/vscode/issues/13920](https://github.com/Microsoft/vscode/issues/13920)

**Comment by rstoenescu** [comment, marked as answer]
This is an error from the CLI connecting to Github to retrieve the starter kit. It's a non-Quasar issue, but a Github server issue or maybe you're behind a proxy?

Alternatively, download the starter kit yourself from https://github.com/quasarframework/quasar-starter-kit:

<img width="832" alt="image" src="https://user-images.githubusercontent.com/4665641/140037693-2b3764ea-6754-43e3-b52c-7680a0022614.png">

Then:

```
$ quasar create <folder> --kit /path/to/unzipped/kit/from/above
```

**Comment by CynthiaBlue** [comment]
Thanks for the replies.  I think it's still a file permission error on my new MacBook.  I'm not behind a Proxy, and according to the text-kit GitHub page they are up and working.

I've tried changing the permissions on my folder & subfolders but I'm still having issues.  Now I'm getting an error when trying to install firebase auth.

**Comment by CynthiaBlue** [comment]
I believe I got this working.

I followed these instructions:
[https://docs.npmjs.com/resolving-eacces-permissions-errors-when-installing-packages-globally](https://docs.npmjs.com/resolving-eacces-permissions-errors-when-installing-packages-globally)

Manually change npm's default directory

**Comment by CynthiaBlue** [comment]
Just a note, I still have problems every time I try to create a new Quasar project.  I get the same error: 
```
Quasar CLI · Failed to download repo quasarframework/quasar-starter-kit#master: EPERM: operation not permitted, utime '/Users/mymac/.quasar-starter-kits/quasarframework-quasar-starter-kit#master/.github'
```

In order to create a new project, I have to download the kit manually and unzip it. Then I have to create my directory, change into it, and run the following code to reference the downloaded and unzipped starter kit:

```
quasar create --kit ../quasar-starter-kit-master
```

Then my project is created properly and runs fine.
